src/components/PageButtons/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+  Button,
+} from '@edx/paragon';
+
+function PageButton(label, props, page) {
```

Comment:
This is not a component in reactjs. We should make sure we use the reactjs component interface. See https://github.com/edx/gradebook/blob/master/src/components/Header/index.jsx#L6 as an example

Reply:
I originally did that, but eslint told me that since this component is stateless it should be a pure function component

Reply:
I'm not sure that the abstraction from `Button` to `PageButton` is actually helpful here yet. I would just put those two buttons as is in the PageButtons component. Then you should destructure props in the function paramater signature. (e.x. `function PageButtons({selectedCohort, prevPage, nextPage, selectedTrack})`)